schemas/events/v1/events.json

```
@@ -95,15 +106,18 @@
     "specversion",
     "type",
     "dataschema",
-    "time",
-    "redhatorgid"
+    "time"
```

the org id is required now

```suggestion
    "time",
    "redhatorgid"
```